brazier.lua

```diff
@@ -1,3 +1,16 @@
+-- check for MineClone2
+local mcl = minetest.get_modpath("mcl_core")
+
+if mcl then
+    item1='mcl_core:torch'
```

Comment:
Can you change the names of the variables `item1`, `item2`, and `item3` to `torch`, `cobble`, and `basic_flame` respectively (in each of the lua files)? That can help to make the source code for the recipes easier to understand at a glance.

Reply:
Updated :)

Reply:
Looks good.